flow-typed/schema.gql

```diff
@@ -10,7 +10,7 @@ type Applicant @model {
   id: ID! @isUnique
   action: Action @relation(name: "ApplicantOnAction")
   applications: [Application!]! @relation(name: "ApplicationOnApplicant")
-  birthdate: String
+  birthDate: String
```

Comment:
I'm not sure what the API is returning but we recently switched to collecting only a birth year if that makes things easier.

Reply:
We just use it for display so if there is a birth year available would be pretty easy to swap that out once it's available in the query.